src/js/themes/hpe.js

```diff
@@ -1824,14 +1825,34 @@ const buildTheme = (tokens, flags) => {
         },
       },
     },
+    nameValueList: {
```

Comment:
Should we also include a re-map for `nameValueList.name.width` from 'small' to 'xsmall'?

Reply:
Done!